static/src/javascripts/projects/commercial/modules/prebid/types.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+export type PrebidIndexExchangeParams = {
+    id: string,
+    siteID: string,
+};
+
+export type PrebidSonobiParams = {
+    ad_unit: string,
+    dom_id: ?string,
+    floor: ?number,
```

Comment:
Revisiting the pain of working on the edge... 🙃 

Does this need to be a maybe type? Should it always be a number?

Reply:
I followed the api docs. These two are genuinely optional, as in you don't need to set a price floor, or pass a dom id, and the bidder will still work.